src/cljs/lomakkeet/reagent/datepicker.cljs

```diff
@@ -57,15 +67,16 @@
           [:input.form-control
```

Comment:
I wonder if the click handler could be attached to DOM here? Probably Pikaday would overwrite it? Another option would be to wrap input in another element, but that might mess up styles?